pkg/kubelet/cm/devicemanager/manager.go

```
@@ -563,11 +563,11 @@ func (m *ManagerImpl) readCheckpoint() error {
 	return nil
 }
 
-// updateAllocatedDevices gets a list of active pods and then frees any Devices that are bound to
-// terminated pods. Returns error on failure.
-func (m *ManagerImpl) updateAllocatedDevices(activePods []*v1.Pod) {
+// UpdateAllocatedDevices frees any Devices that are bound to terminated pods. Returns error on failure.
+func (m *ManagerImpl) UpdateAllocatedDevices() error {
```

I don't think we need to return an error here.